src/main/java/com/banking/bankservice/service/UserService.java

```diff
@@ -0,0 +1,16 @@
+package com.banking.bankservice.service;
+
+import com.banking.bankservice.model.User;
+import java.util.Optional;
+
+public interface UserService {
+    User create(User user);
+
+    User update(User user);
+
+    Optional<User> getById(Long id);
+
+    Optional<User> getByPhone(String phone);
```

Comment:
let's return entity from the service layer

Reply:
refactored